src/index.html

```diff
@@ -1,13 +1,66 @@
-<!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<!DOCTYPE html>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="page__content">
+
+    <div class="card" data-qa="card">
+      <div class="card__inner">
+        <div class="card__img">
+          <img src="images/imac.jpeg" alt="product">
+        </div>
+
+        <h2 class="title card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+        <div class="card__product">Product code: 195434</div>
+
+        <div class="card__star review">
+          <ul class="stars review__stars review__stars--4">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </ul>
+          <div class="review__reviews">Reviews: 5</div>
+        </div>
+
+        <div class="card__price price">
+          <span class="price__title">Price:</span>
+          <strong class="price__price">$2,199</strong>
+        </div>
+
+        <div class="card__action action">
```

Comment:
check this wrapper, you can add all needed styles to the link and don't create additional blocks